tests/integration/docker/volume.bats

```diff
@@ -48,6 +48,31 @@ setup() {
 	[ $RC -eq 0 ]
 }
 
+@test "Volume - bind-mount a directory" {
+	dir_path="$(pwd)/sharedSpace"
+	test_file="foo"
+	containerPath="/root/sharedSpace"
+	mkdir "$dir_path"
```

Comment:
seems like bats `run` command has issues when using it with pipes `|`, need to find another way.